tests/terraform/checks/provider/aws/test_defaulttags.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from checkov.terraform.checks.provider.aws.defaulttags import check
+from checkov.common.models.enums import CheckResult
+
+
+class TestDefaultTags(unittest.TestCase):
+
+    def test_success(self):
```

Comment:
This needs to use the test format described here: https://github.com/bridgecrewio/checkov/blob/master/CONTRIBUTING.md#tests-for-new-checks

This is the way it says NOT to do it

Reply:
@mikeurbanski1 This should be resolved using the alternate test method you suggested.